### PR TITLE
[fibrechannel] Add new plugin

### PR DESCRIPTION
**Pull request by TurboTurtle**
Adds a new plugin for fibrechannel devices.

Signed-off-by: Jake Hunsaker <jhunsake@redhat.com>

---
Please place an 'X' inside each '[]' to confirm you adhere to our [Contributor Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines)

- [x] Is the commit message split over multiple lines and hard-wrapped at 72 characters?
- [x] Is the subject and message clear and concise?
- [x] Does the subject start with **[plugin_name]** if submitting a plugin patch or a **[section_name]** if part of the core sosreport code?
- [x] Does the commit contain a **Signed-off-by: First Lastname <email@example.com>**?

### CODE REVIEW

File: sos/plugins/__init__.py [hunk 1]
```diff
@@ -860,6 +860,26 @@ def add_journal(self, units=None, boot=None, since=None, until=None,
         self._log_debug("collecting journal: %s" % journal_cmd)
         self._add_cmd_output(journal_cmd, None, None, timeout)
 
+    def add_udev_info(self, device, attrs=False):
```
Comment (bmr-cymru) [inline sos/plugins/__init__.py]: This is a big new interface in the `Plugin` class: it deserves its own commit, and description.

Comment (bmr-cymru) [reply sos/plugins/__init__.py]: It might be good to also support `-p` so that we can directly replace existing `add_cmd_output()` calls for `udevadm` with this method. This also needs some thought as to how it's going to function in a container - is `/sys` mounted where we'd expect?

**Comment by TurboTurtle** [comment]
Ack. I split the `add_udev_info()` addition into a new PR, #1293.

As far as the RH support team has told me, there isn't anything else they need to see in a sosreport beyond what's already captured (lspci, etc...) once this plugin exists.
